### PR TITLE
Create new Esperanto cheat sheet

### PR DESCRIPTION
**Pull request by abadojack**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [* ] Cheat Sheets: **`New Esperanto Cheat Sheet`**
    
###### Description of changes
Provides a cheat sheet for the Esperanto language.

Instant Answer Page: https://duck.co/ia/view/esperanto_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @abadojack

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/esperanto.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
+{
+   "id" : "esperanto_cheat_sheet",
+   "name" : "Esperanto Cheat Sheet",
+   "metadata" : {
+      "sourceName" : "Linguanaut",
+      "sourceUrl" : "http://www.linguanaut.com/english_esperanto.htm"
+   },
+   "aliases":[
+		"esperanto words", "esperanto greetings"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/esperanto.json]: lets add some more aliases.

**Comment by zekiel** [comment]
I think this is failing bc `Sections` doesn't match the ordering you put them in. I can't spot anything else from this glance.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Esperanto cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/esperanto_cheat_sheet_mobile.png?fpZuPJ8xKA)

**Comment by rasikapohankar** [comment]
@zekiel I looked at few live cheat sheets, the order in `Sections` can be different from the one in `section_order`. The test has failed saying `No Instant Answer page found with ID 'esperanto_cheat_sheet'`. Perhaps its a metadata issue as @GuiltyDolphin had mentioned, I am not sure how it works internally.

**Comment by abadojack** [comment]
@rasikapohankar I already created an Instant Answer page with the id 'esperanto_cheat_sheet'. @GuiltyDolphin Why is the build failing ?

**Comment by GuiltyDolphin** [comment]
@abadojack Build just needed re-running :+1:

**Comment by zekiel** [comment]
Thanks @abadojack -- merging this.

**Pull request merged**